OneCommunityGlobal/HGNRest
# yihan create way to see 6 months & yearly anniversaries

# Description
![image](https://github.com/OneCommunityGlobal/HGNRest/assets/94303659/c77138f6-6f72-4985-b7f1-46eaa8e4f9a0)

## Related PRS (if any):
To test this backend PR you need to checkout the [#1603](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1603) frontend PR.

## Main changes explained:
- Update file dashBoardController.js and dashboardRouter.js for including a function of posting trophy data.
- Update file dashboardhelper.js, reporthelper.js and userProfile.js for adding trophy icon property

## How to test:
1. check into current branch on both FE and BE
2. do `npm install` and `...` to run this PR locally
3. follow the test steps of [#1603](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1603).
